packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -1434,26 +1464,21 @@ describe(componentName, () => {
     const rowExpander = row.querySelector(`button[aria-label="Expand row"]`);
     fireEvent.click(rowExpander);
 
-    expect(
-      screen
-        .getByRole('table')
-        .getElementsByTagName('tbody')[0]
-        .getElementsByClassName('c4p--datagrid__expanded-row')
-    ).toBeDefined();
-    expect(
-      screen
-        .getByRole('table')
-        .getElementsByTagName('tbody')[0]
-        .getElementsByClassName('c4p--datagrid__expanded-row')[0].textContent
-    ).toEqual(`Content for ${rowNumber}`);
+    expect(row.nextElementSibling).toHaveClass(`${blockClass}__expanded-row`);
+    expect(row.nextElementSibling.textContent).toEqual(
+      `Content for ${rowNumber}`
+    );
+
+    fireEvent.mouseOver(row.nextElementSibling);
+    fireEvent.mouseLeave(row.nextElementSibling);
```

Comment:
These events are firing, but it's not obvious to me what they're testing for.

Reply:
Removed for now since I wasn't able to confirm the exact class name is being added. I'll look into this later on after the initial base component is released since this is part of expandable rows (scheduled for first extension release, not part of base component).